src/devices/uFireIse/UFirePh .cs

```diff
@@ -0,0 +1,169 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Device.I2c;
+using UnitsNet;
+
+namespace Iot.Device.UFire
+{
+    /// <summary>
+    /// Get pH measuremens from μFire ISE (ion-selective electrode) Probe Interface
+    /// </summary>
+    public class UFirePh : UFireIse
+    {
+        private const float PROBE_MV_TO_PH = 59.2F;
```

Comment:
As for all the other constants which are not really defined like this in the doc, please use the proper notation. So would be ProbeMvToPh
TempCorrectionFactor